code-challenges/401/arrayReverse/array-reverse.js

```diff
@@ -0,0 +1,18 @@
+// Write a function that takes an array as an argument. Without utilizing any of the methods in my language, return an array with elements in reverse order.
```

Comment:
This needs to be a function, not free code.

Reply:
Thank you for the insight, Calvin.